.vitepress/config.ts

```diff
@@ -56,6 +56,7 @@ const sidebars = (): DefaultTheme.SidebarItem[] => [
         link: '/docs/getting-started/supabase-functions',
       },
       { text: 'Node.js', link: '/docs/getting-started/nodejs' },
+      { text: 'Ali Function Compute', link: '/docs/getting-started/ali-function-compute' },
```

Comment:
This is my preference, but I prefer making `Node.js` at the bottom and moving "Ali Function Compute" above `Node.js`. Can you do that?

Reply:
fixed by <https://github.com/honojs/website/pull/417/commits/1666317e3093b6733a1a7077a1ed2bb096971e16>

Reply:
fixed by <https://github.com/honojs/website/pull/417/commits/868aa55d5b07ec001e210dc99b3628aafeebbc8d>

Reply:
fixed by <https://github.com/honojs/website/pull/417/commits/1666317e3093b6733a1a7077a1ed2bb096971e16>